ui/js/controllers/perf/e10s-trend.js

```diff
@@ -0,0 +1,568 @@
+"use strict";
+
+perf.value('defaultBaseDate', 604800);
+perf.value('defaultNewDate', 0);
+perf.value('defaultSampleSize', 604800);
+
+perf.controller('e10sTrendCtrl', [
+    '$state', '$stateParams', '$scope', '$rootScope', '$q', '$http', '$httpParamSerializer',
+    'ThRepositoryModel', 'ThResultSetModel', 'PhSeries', 'PhCompare',
+    'thServiceDomain', 'thDefaultRepo', 'phSampleSizes', 'defaultSampleSize',
+    'phDatePoints', 'defaultBaseDate', 'defaultNewDate',
+    function e10sTrendCtrl($state, $stateParams, $scope, $rootScope, $q, $http, $httpParamSerializer,
+                      ThRepositoryModel, ThResultSetModel, PhSeries, PhCompare,
+                      thServiceDomain, thDefaultRepo, phSampleSizes,
+                      defaultSampleSize, phDatePoints, defaultBaseDate, defaultNewDate) {
+        var blockers = {
```

Comment:
Since this is shared between the e10s dash and the new dash, we should probably put it in values.js.